beacon-chain/core/epoch/epoch_processing_test.go

```diff
@@ -1103,6 +1124,95 @@ func TestAttestationDelta_SomeAttested(t *testing.T) {
 				0, penalties[i])
 		}
 	}
+
+	nonAttestedIndices := []uint64{12, 23, 45, 79}
+	for _, i := range nonAttestedIndices {
```

Comment:
This seems to be repeated - maybe abstract into a common function for the tests?

Reply:
I think it's fine to leave them separate, too many outside vars being used here and the above code is slightly different.